common/charts/couchdb/values.yaml

```diff
@@ -54,6 +54,12 @@ helperImage:
 ## `OrderedReady`
 podManagementPolicy: Parallel
 
+
+## To better tolerate Node failures, we can enable Kubernetes scheduler to not assign
+## more than one Pod of CouchDB StatefulSet per Node using podAntiAffinity.
+## This is still a beta feature, and will get stable in Kubernetes 1.2
```

Comment:
`Kubernetes 1.2` is probably the wrong version! 1.2 looks like the version that affinity was introduced. Definitely a case of the documentation lying to the reader.

Reply:
Nice catch!